modulefiles/obs-monitor/ursa.lua

```diff
@@ -0,0 +1,54 @@
+help([[
+Load python virtual environment for obs-monitor
+]])
+
+local pkgName    = myModuleName()
+local pkgVersion = myModuleVersion()
+local pkgNameVer = myModuleFullName()
+
+conflict(pkgName)
+
+
+prepend_path("MODULEPATH", "/contrib/spack-stack/spack-stack-1.9.2/envs/ue-oneapi-2024.2.1/install/modulefiles/Core")
+
+load("stack-oneapi/2024.2.1")
+load("stack-intel-oneapi-mpi/2021.13")
+load("intel-oneapi-mkl/2024.2.1")
+load("stack-python/3.11.7")
+load("rocoto/1.3.7")
+
+load("py-jinja2/3.1.4")
+load("py-netcdf4/1.7.1.post2")
+load("py-pybind11/2.13.5")
+load("py-pycodestyle/2.11.0")
+load("py-pyyaml/6.0.2")
+load("py-scipy/1.14.1")
+load("py-xarray/2024.7.0")
+load("py-pip/23.1.2")
+load("py-matplotlib/3.7.4")
+load("py-cartopy/0.24.1")
+load("proj/9.4.1")
+load("py-wxflow/0.2.0")
+
+local pyenvpath = "/scratch3/NCEPDEV/da/Edward.Safford/noscrub/python/envs/"
+local pyenvname = "obs-mon"
+
+local pyenvactivate = pathJoin(pyenvpath, pyenvname, "bin/activate")
+
+if (mode() == "load") then
+  local activate_cmd = "source "..pyenvactivate
+  execute{cmd=activate_cmd, modeA={"load"}}
+  prepend_path("PATH", "/scratch3/NCEPDEV/da/Edward.Safford/noscrub/python/envs/obs-mon/bin")
+  prepend_path("PYTHONPATH", "/scratch3/NCEPDEV/da/Edward.Safford/noscrub/python/envs/obs-mon")
+
+else
```

Comment:
One of my AI queries a few days ago mentioned that instead of `activate` and `deactivate` that there was a better way to use a python virtual environment when loading a module but I forget the details. May be worth asking about it to a "friend"